crates/ruff/src/rules/pycodestyle/rules/errors.rs

```diff
@@ -7,6 +7,27 @@ use ruff_source_file::Locator;
 
 use crate::logging::DisplayParseErrorType;
 
+/// ## What it does
+/// This is not a regular lint, instead it is raised when a file can not be read.
+///
+/// ## Why is this bad?
+/// This indicates an error in the development setup. A common cause of IO errors is that the
+/// current user doesn't have the permission to read the file. Another possible reason is a broken
+/// symlink.
+///
+/// ## Example
+/// On linux:
```

Comment:
Should this be `unix` or `linux and Mac OS` instead?

Reply:
Yeah maybe like `Linux or macOS`?

Reply:
TIL they switched from Mac OS X to macOS in 10.12